tradermade/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@chainlink/tradermade",
```

Comment:
Isn't it `@chainlink/tradermade-adapter` now?

Reply:
Good point

Reply:
Oh actually, we can't do this until the other PR is merged in. Reverting that commit now

Reply:
Ok, let's get that other PR rebased & merged after this then.